Framework/PythonInterface/mantid/_testing/AssertAlmostEqual.py

```diff
@@ -0,0 +1,48 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2023 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from mantid.simpleapi import CompareWorkspaces
+
+
+def assert_almost_equal(Workspace1, Workspace2, rtol=0.0, atol=0.0):
```

Comment:
unfortunately this is very misleading to me because it reminds me of Numpy's `assert_almost_equal` but this function doesn't behave like Numpy's. For instance, a user typing `assert_almost_equal(Workspace1, Workspace2, rtol=0.9, atol=6.0)` may not be aware that `atol=6.0` is ignored!
I rather amend `CompareWorkspaces` with backwards compatibility to avoid the previous scenario